src/hostCommands/testCompile.ts

```diff
@@ -119,6 +125,38 @@ function parseCompilerOutput(compilerOutput: string, document: vscode.TextDocume
 	return pslDiagnostics;
 }
 
+function canTestCompileFile(document: vscode.TextDocument, fsPath: string) {	
+	let compilable: boolean = false;
+	if (vscode.languages.match(extension.PSL_MODE, document)) {
+		compilable = true;
+	}
+	else {
+		let fileTypeDescription = "";
+		if (vscode.languages.match(extension.BATCH_MODE, document)) {
+			fileTypeDescription = "batches"
+		}
+		else if (vscode.languages.match(extension.COL_MODE, document)) {
+			fileTypeDescription = "column definitions"
+		}
+		else if (vscode.languages.match(extension.DATA_MODE, document)) {
+			fileTypeDescription = "data files"
+		}
+		else if (vscode.languages.match(extension.TBL_MODE, document)) {
+			fileTypeDescription = "table definitions"
+		}
+		else if (vscode.languages.match(extension.TRIG_MODE, document)) {			
+			fileTypeDescription = "triggers"
+		}
+		if (fileTypeDescription != "") {
+			utils.logger.error(`${utils.icons.ERROR} ${icon} ${fileTypeDescription} cannot be test compiled (${path.basename(fsPath)}).`);
```

Comment:
What if you change the plurality of `fileTypeDescription` to singular and also capitalize it? Then you can rewrite the error message like `${utils.icons.ERROR} ${icon} ${fileTypeDescription} ${path.basename(fsPath)} cannot be test compiled.` 

For example:
`batches cannot be test compiled (BTTDRV.BATCH)`
versus
`Batch BTTDRV.BATCH cannot be test compiled`

Which do you prefer?